HealthcareSystem/PatientWebApp/wwwroot/js/patient_registration.js

```diff
@@ -0,0 +1,27 @@
+﻿$(document).ready(function () {
+
+	$('#image_row').empty();
+
+	/* Display the image on the html page */
+	var chosen_image = [];
+	if (window.File && window.FileList && window.FileReader) {
+
+		$("#add_image").on("change", function (e) {
```

Comment:
Is 'e' event?

Reply:
> Is 'e' event?

Yes. Is it necessary to change that name?

Reply:
Maybe no, that is good. Problem is that I'm not good enough on JS, because of that I ask.